README.md

```diff
@@ -1,2 +1,79 @@
 # product-landing-page
 A product landing page about GUCCI;
+ #Project Title 🚀
+Gucci landing page
+
+## Project Description 📝
+
+This is a product landing page about Gucci italian luxury fashion brand.This page contains the HTML and CSS.
+
+## Demo 📸
+https://www.linkedin.com/in/farhat-sharefi-13a101309?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app
+
+## Technologies Used 🛠️
+
+
+- HTML
+- CSS
+
+## Installation 💻
+
+first of all you should install these apps:
+github
+gitbash
+freecodecamp
+
+after that u have to log in to your own account or sign up by creating a new account.
+
+also seeing my project you have to clone my prokect in your local machine.
+this is my ssh you can copy and paste in your github
+git@github.com:iamfarhatsharefi/product-landing-page.git
+-freecodecamp is for runing my code that yoy can be sure that the project is correct.
+
```

Comment:
Instead of this just give user a hint and a command for cloning your project and opening it on browser.